Source/Relativity.Test.Helpers/Relativity.Test.Helpers/ArtifactHelpers/FieldsHelper.cs

```diff
@@ -81,61 +81,61 @@ public static int CreateField(IRSAPIClient client, FieldRequest request)
 			//Set the workspace ID
 			client.APIOptions.WorkspaceID = request.WorkspaceID;
 			//Create a Field DTO
-			var fieldDTO = new DTOs.Field();
+			DTOs.Field fieldDTO = new DTOs.Field();
 			//Set secondary fields
 			request.HydrateFieldDTO(fieldDTO);
 			//Create the field
-			var resultSet = client.Repositories.Field.Create(fieldDTO);
+			DTOs.WriteResultSet<DTOs.Field> resultSet = client.Repositories.Field.Create(fieldDTO);
 			//Check for success
 			if (resultSet.Success)
 			{
 				fieldID = resultSet.Results.FirstOrDefault().Artifact.ArtifactID;
 			}
 			else
 			{
-				var innEx = resultSet.Results.Any() ? new Exception(resultSet.Results.First().Message) : null;
+				Exception innEx = resultSet.Results.Any() ? new Exception(resultSet.Results.First().Message) : null;
```

Comment:
we are throwing general exception here. update to custom exception.

Reply:
Done